libutils/string_lib.h

```diff
@@ -59,7 +59,12 @@ char ToUpper(char ch);
 void ToUpperStrInplace(char *str);
 void ToLowerStrInplace(char *str);
 
-long StringToLong(const char *str);
+int StringToLong(const char *str, long *number) FUNC_WARN_UNUSED_RESULT;
+void LogConversionError(const char *str_attempted, const char *id);
```

Comment:
Why export this function. Do you use it in nova? If not, why even have a function?

Reply:
Yes it is used in nova. To avoid code duplication.